core/algorithms/simplify.py

```diff
@@ -1153,7 +1181,7 @@ def simplify_clusters(artifacts, roads, max_segment_length=1, eps=1e-4):
     # global and this step should happen only once, not for every artifact
     new = gpd.GeoDataFrame(geometry=to_add, crs=roads.crs)
     new["_status"] = "new"
-    new["geometry"] = new.line_merge().simplify(max_segment_length)
+    new["geometry"] = new.line_merge().simplify(max_segment_length * 2)
```

Comment:
What's significance of `* 2` here? Is this adding some tolerance? Same with above in L129.

Reply:
This ensures elimination of unnecessary angles within the skeleton-based LineStrings beyond smoothing of the Voronoi-based artifacts. I forgot to finish this part, this was to try that and I wanted to expose that as a parameter rather than hard-coding it (though 2 seems to be a decent default). Can expose it in a follow-up.

Reply:
A follow-up PR or commit?